src/Core/Silk.NET.SilkTouch/NativeApiAttribute.cs

```diff
@@ -0,0 +1,47 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Silk.NET.Core.Native
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method | AttributeTargets.Property)]
+    public class NativeApiAttribute : Attribute
```

Comment:
Couldn't you just reference Silk.NET.Core and use the real NativeApiAttribute to prevent code duplication?

Reply:
Unfortunately not, I have no clue how project references are supposed to work with source generators. Just referencing doesn't work

Reply:
Damn that sucks. An alternative in that case could be adding the following item to an ItemGroup in the csproj:
```xml
<Compile Include="..\Silk.NET.Core\Native\NativeApiAttribute.cs" />
```